Language/Functions/Interrupts/interrupts.adoc

```diff
@@ -17,10 +17,9 @@ subCategories: [ "Interrupts" ]
 
 [float]
 === Description
-Re-enables interrupts (after they've been disabled by link:../nointerrupts[noInterrupts()]. Interrupts allow certain important tasks to happen in the background and are enabled by default. Some functions will not work while interrupts are disabled, and incoming communication may be ignored. Interrupts can slightly disrupt the timing of code, however, and may be disabled for particularly critical sections of code.
+Re-enables interrupts (after they've been disabled by link:../nointerrupts[noInterrupts()]. Interrupts allow certain important tasks to happen in the background and are enabled by default. Some functions will not work while interrupts are disabled, and incoming communication may be ignored. Interrupts can slightly disrupt the timing of code, however, and may be disabled for particularly critical sections of code. They can also cause races, code that behaves differently, and incorrectly, in response to tiny changes in timing. See the link:../External%20Interrupts/attachInterrupt[attachInterrupt()] page for further information.
 [%hardbreaks]
 
-
```

Comment:
Let's leave the line break here because it is unrelated to the proposal and this is the style established in the reference sample:

https://raw.githubusercontent.com/arduino/reference-en/master/AsciiDoc_sample/Reference_Terms/AsciiDoc_Template-Single_Entity.adoc